### ISSUE TITLE
Explicitly disable merged mining

### ISSUE THREAD

**Issue by jagerman** [loki-project/loki#620]
I'd like to propose that in the next fork we forcibly disallow merge mining by requiring that coinbase transactions in the next network version do not include coinbase transactions with merge mining tags (i.e. the tx extra field TX_EXTRA_MERGE_MINING_TAG `0x03`).

Merged mining has been, on the whole, a fairly undesirable feature.  Although it wasn't intentional, it ended up putting burdens on pools to support it, and (early on) had some pools throwing out dirty hacks in order to get it working: one pool stopped including any transactions in blocks; another made the coinbase transaction version 1, which resulted in the resulting block rewards being unspendable (for the pool but also SNs and the governance wallet).

The tradeoff has been, of course, a significant hashrate increase.  With the upcoming checkpointing support, however, this hashrate is essentially pointless: we don't *need* it to secure the network against 51% attacks.  Instead that role now moves to service node checkpointing quorums, which are considerably more difficult (and expensive) to take control of than a short-term 51% mining attack has been.

Thus this issue to discuss the matter.

**Comment by necro-nemesis** [comment]
Concur. It further highlights the strength of the project and the efforts put into development of check pointing and the node network. This is one aspect where Loki core shines. Merge mining should be a concern of other projects and not Loki as Loki has diligently worked to overcome shortfalls others are willing to compromise and accept. Go Loki core!

**Comment by Dormage** [comment]
I support the idea. Aside from the technical issues one can argue merged mining is in part devaluing Loki. 

Initially merged mining was very profitable. However, through economics, the system stabilized the value across both chains resulting in a significantly lower valuation of Loki.

As a result, miners that supported the network from the very beginning and held Loki as a speculative investment (rather then selling to cover the costs of mining) are now having their investment devalued.

**Comment by KeeJef** [comment]
I disagree with this change atleast until checkpointing commences, at this point our hashrate is exceedingly well distributed across a number of pools, with no pool having a  majority hashrate or even close to it https://miningpoolstats.stream/loki

I would attribute most of this to merge mining with turtlecoin which is right now providing Loki double the security and a more decentralized mining pool distribution, Turtle will fork to thier new Argon2 based hashing algorithm at block 1,800,000 with a current block height on turtlecoin of 1,588,887 and a blocktime of 30s we should see the fork happen some time around ~August the 29th. If everything goes well for checkpointing we are scheduled for an activation date of ~September the 12th. this will mean we will maintain a high degree of protection by combining turtlecoin hashrate and Loki hashrate till that point.

Once checkpointing is enabled im open to the idea of disabling Merge mining, however i think its somewhat unclear what the risk/benefits are longterm of enabling merged mining are. For a good example of where merge mining has worked for security we can look at Namecoin which about 90% of Bitcoin miners are mining https://bitinfocharts.com/comparison/hashrate-btc-nmc.html

If the Loki community agreed to reduce the block reward percentage allocated to Miners and still requires miners to create blocks and derive randomness (one option). then it would be good to maintain a high and stable hashrate, which could be achieved by pairing with a parent coin to derive hashrate.

**Comment by bobbieltd** [comment]
Thanks to your opinion, I got my lesson. A complete 180.
There are many options to deal with merged mining but you propose the radical one (if we do not like it, we kill it).

**Comment by Dormage** [comment]
> If the Loki community agreed to reduce the block reward percentage allocated to Miners and still requires miners to create blocks and derive randomness (one option). then it would be good to maintain a high and stable hashrate, which could be achieved by pairing with a parent coin to derive hashrate.

I think this would work. Also, AFAIK, the initial idea was to change this after check-pointing is implemented. Arguably, merged mining has increased network security but I would argue it has also effected the price action considerably. This has effected service node operators who are not struggling even to cover the costs of hosting. 

Full disclosure, I operate 7 service nodes.

**Comment by SomethingGettingWrong** [comment]
n the case of decentralized emission.... when you have a single entity thats 75 percent more profitable it centralizes the emission and does the exact same thing as a premine. so its imperative a coin does something exp on a coin like loki... if hes is allowed 75 percent for an x amount of time.. then stakes it.. anonyomity of the service nodes is gone and he will forever have x amount of emission right now the turtlecoin is being mergemined and the hashrate is not securing anything on the network once checkpoints are implemented.  majority of the hashrate is FPGA
